torch/ao/quantization/fx/prepare.py

```diff
@@ -221,14 +230,16 @@ def is_observer_in_same_graph(node, modules, node_name_to_target_dtype):
             return False
     return True
 
-def is_pattern_dtype_config_supported_by_backend(
+def _is_pattern_dtype_config_and_qconfig_supported_by_backend(
     pattern: Optional[Pattern],
-    matched_node_pattern: Optional[NodePattern],
+    matched_node_pattern: Optional[List[Node]],
     node_name_to_target_dtype: Dict[str, Dict[str, Optional[Tuple[Union[torch.dtype, type], bool]]]],
+    qconfig: QConfigAny,
     backend_config: BackendConfig,
 ) -> bool:
     """ Check is the dtype configuration of a pattern is supported by
-    the backend or not
+    the backend or not. Also validate whether the qconfig satisfies
```

Comment:
nit: update the comment as well since `validate` means we'll error out when something doesn't match